### ISSUE TITLE
Black screen mobile camera is not opening 

### ISSUE THREAD

**Issue by ghazali112** [nakulkundaliya/ionic-3-video-calling-using-webrtc#9]
when i try to call from one mobile to another then only auido is working but video is not showing black screen appear in video portion it works perfectly on web browser but after building it for android its not showing video of my camera its only showing video of other web browser call .

**Comment by Saqib92** [comment]
Same issue here.

**Comment by ghazali112** [comment]
i ressolve this issue by providing permission from my mobile go to settings
go to permission and allow camera from there

**Comment by Saqib92** [comment]
I did the Same as you did. Problem Solved by given camera/ all permissions.

**Comment by ghazali112** [comment]
yes but this api is not usefull for social platform because it contain id i
want video calling like facebook messenger and watsapp

**Comment by Saqib92** [comment]
for this you have to save that ID into mobile + database and call against this ID.

**Comment by ghazali112** [comment]
the id is changing after every refresh of page problem is that

**Comment by Saqib92** [comment]
You have to save ID generated First time and stop regenerating it.
